__tests__/__fixtures__/MonographInstance.json

```diff
@@ -133,12 +133,7 @@
       "resourceTemplates": [],
       "type": "resource",
       "valueConstraint": {
-        "defaults": [
-          {
-            "defaultLiteral": "DEFAULT",
-            "defaultURI": "http://default"
-          }
-        ],
+        "defaults": [ ],
```

Comment:
It doesn't make sense to have defaults and valueTemplateRefs, so I chose to eliminate the default.

Reply:
I'm not seeing `valueTemplateRefs` here, so I'm confused by your comment.  And it wouldn't surprise me if some tests depend on the defaults here.